sympy/logic/boolalg.py

```diff
@@ -16,9 +16,9 @@
 from sympy.core.singleton import Singleton, S
 from sympy.core.sorting import ordered
 from sympy.core.sympify import _sympy_converter, _sympify, sympify
-from sympy.utilities.iterables import sift, ibin
+from sympy.utilities.iterables import sift, ibin, iterable
 from sympy.utilities.misc import filldedent
-
+import operator
 
```

Comment:
add another line here

Reply:
Above 21 or below 21 ?

Reply:
below 21

Reply:
functions must be separated with two lines.